### DISCUSSION TITLE
Regular Expression 的翻譯

### DISCUSSION THREAD

**Discussion by ascodeasice** [python/python-docs-zh-tw#638] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

Regular Expression

### 術語翻譯

正規表示式

## 原因

> 據我所知 regex 的中文翻譯有各種組合`(正規|正則|常規|規則)(((表達|表示)(法|式))|運算式)`，那可以請你幫忙思考/調查`正規表示式`是否為最合適的選擇嗎？
也請考慮到目前我們術語表已將 expression 統一翻譯為 運算式。
By [mattwang44](https://github.com/mattwang44)

[原 comment 連結](https://github.com/python/python-docs-zh-tw/pull/635#issuecomment-1702165999)

## 主觀想法

常規與規則這兩種前綴幾乎不會看到，所以不考慮。
運算式是目前 expression 的統一翻譯，但是 regular expression 翻譯成＿＿運算式很罕見，所以主觀認為不適合。
＂法＂結尾的翻譯也很少見。
所以如果只考慮常見的翻譯，只剩下 (正則|正規)(表達|表示)式

個人覺得正規/正則表示式念起來比較順

## 其他資料

- [舊版 (3.11) 繁中 python 文件](https://docs.python.org/zh-tw/3.11/library/re.html)使用正規表示式
- [舊版 (3.11) 簡中 python 文件](https://docs.python.org/zh-cn/3.11/library/re.html)使用正則表達式

以上多為個人看法，我覺得正規、正則，表示、表達其實可以互通，主要是為了方便所以統一翻譯。
個人覺得可以依照以前的文件翻成正規表示式就好，或是可以不翻（但提供讀者一些常見的翻譯方法）
想問各位覺得如何翻比較好

**Comment by cschan1828** [comment]
正規表示式 +1

**Comment by mattwang44** [comment]
「常規表達式/正則表達式」

https://breezymove.blogspot.com/2023/11/regular-expression.html
> 至於常聽到的「正規表達式」，正規是錯譯。中文的正規代表正式規範，意即英文的 Norm/Normal。
